galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml

```diff
@@ -1333,7 +1333,103 @@ paths:
               schema:
                 $ref: '#/components/schemas/JsonError'
 
-
+  #--------------------------------------
+  # Update the status of an active test 
+  # run to reset or delete it.
+  #--------------------------------------
+    put:
```

Comment:
The reason we have gone with PUT for both is because it is essentially a status update. It would definitely help, however I am concerned that using delete with the API as is may cause confusion as to whether or not it will delete a completed run as well.

I think something like this should be highlighted in our design session for the APIs, so we have an agreed approach for all endpoints